cpp/src/arrow/chunked_array.h

```diff
@@ -36,6 +36,10 @@ namespace arrow {
 class Array;
 class DataType;
 class MemoryPool;
+namespace stl {
+template <typename T, typename V>
+class ChunkedArrayIterator;
+}
```

Comment:
weird, doesn't the linter usually require `}  // namespace stl` here?

Reply:
It should indeed.